### PR TITLE
LPS-151540 Set current values to object entry

### PR DESCRIPTION
**Pull request by seltong**
Related issue: https://issues.liferay.com/browse/LPS-163725

Follow up - Brian: https://github.com/brianchandotcom/liferay-portal/pull/125129

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:reevaluate:1357401_2929

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
@seltong could you rebase and check if the test failures are related? Thx.

**Comment by seltong** [comment]
@gabrielwas I will take a look at this as soon as possible

**Comment by seltong** [comment]
Rebase and Semantic versioning fix

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectActionVariablesUtil.java [hunk 1]
```diff
@@ -36,6 +40,59 @@ public static Map<String, Object> toVariables(
...
+				HashMapBuilder.<String, Object>put(
+					"creator", payloadJSONObject.get("userId")
+				).build();
+
+			Map<String, Object> allVariables = new HashMap<>();
+
+			if (objectDefinition.isSystem()) {
+				String contentType = _getContentType(
+					dtoConverterRegistry, objectDefinition,
+					systemObjectDefinitionMetadataTracker);
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectActionVariablesUtil.java:54]: Move this variable to closer of its usage

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java [hunk 2]
```diff
@@ -185,40 +193,60 @@ public ObjectValidationRule updateObjectValidationRule(
...
+			objectValidationRuleLocalService.getObjectValidationRules(
+				objectDefinitionId, true);
 
-		if (baseModel instanceof ObjectEntry) {
-			values = HashMapBuilder.<String, Object>putAll(
-				baseModel.getModelAttributes()
-			).putAll(
-				_objectEntryLocalService.getValues((ObjectEntry)baseModel)
-			).build();
+		if ((objectValidationRules == null) ||
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java:209]: Not need `objectValidationRules == null`

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:reevaluate:1347401_3109

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by pedro-michel** [comment]
ci:stop

**Comment by seltong** [comment]
ci:help

**Comment by seltong** [comment]
Resent here: https://github.com/liferay-objects/liferay-portal/pull/1314

**Pull request closed without merging**